tests/test_cli.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+import sys
+from contextlib import contextmanager
+from textwrap import dedent
+
+from django.core.management import call_command, CommandError
+from django.test import TransactionTestCase
+from django.utils.six import StringIO
+
+from constance import config
+
+@contextmanager
+def redirect_stdout(new_target):
```

Comment:
Hm, what does Django do for this? Does it contain a helper we can reuse to test the out of the management command?